Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -46,7 +72,6 @@ public void SetMasterVolume(float volume)
             PlayerPrefs.Save();
         }
 
-        /// <summary> Sets BGM volume (0.0001 to 1). </summary>
```

Comment:
nit: this doc was removed

also github no allow me to select code that didn't change so i have to select each one manually